python/repositories.bzl

```diff
@@ -104,6 +111,12 @@ def _python_repository_impl(rctx):
             fail(exec_result.stderr)
 
     python_bin = "python.exe" if ("windows" in platform) else "bin/python3"
+    print(native.bazel_version)
+    if _parse_native_bazel_version(native.bazel_version) >= (5,0,0):
+        abs_python_bin_path = rctx.path(python_bin)
+        stub_shebang_assignment = "stub_shebang = \"#!{}\"".format(abs_python_bin_path)
```

Comment:
```suggestion
    if _parse_native_bazel_version(native.bazel_version) >= (5,0,0):
        execroot = "/".join(str(rctx.path(".")).replace("\\", "/").split("/")[:-2])
        python_bin_path = str(rctx.path(python_bin)).removeprefix(execroot + "/")
        shebang_line = "#!./" + python_bin_path
        stub_shebang_assignment = "stub_shebang = '{}',".format(shebang_line)
```

Reply:
This will find the binary relative to the execroot.

Reply:
I tried something similar, unfortunately this does not work for me (bazel 5.1.0, rules_python 0.9.0) as the python repository is not part of the linux sandbox dependencies when running the `py_binary()`.

Did you run this successfully?